Makefile

```diff
@@ -5,6 +5,9 @@ GOPATH=$(shell go env GOPATH)
 GOBIN=$(GOPATH)/bin
 GOSRC=$(GOPATH)/src
 
+MKDOCS_IMAGE := aquasec/mkdocs-material:starboard
```

Comment:
NIT You may want to rename tag to dev or trivy

Reply:
My bad... Done https://github.com/aquasecurity/trivy/pull/884/commits/098a322a049f45a29be462df8aa136410facfb1b